### ISSUE TITLE
Update User registration component to send correct JSON structure for account upgrade request

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/whispers#921]
Current sending:

```
{
  "username": "testPUBlake1",
  "first_name": "test",
  "last_name": "PUBlake1",
  "email": "bdraper+6@usgs.gov",
  "password": " testpublak10@",
  "organization": 1737,
  "role": 3,
  "message": "please let me have this"
}
```
Should be:
```
{
    "username": "someguy14",
	  "password": "abcdefg1234567!",
    "first_name": "Some",
    "last_name": "Guy",
    "email": "someguy14@example.biz",
    "groups": [],
    "user_permissions": [],
    "is_superuser": false,
    "is_staff": false,
    "is_active": true,
	  "new_user_change_request": {
			"role_requested": 5,
			"organization_requested": 1066,
			"comment": "Hello yes please."
		}
}
```

**Comment by BlakeDraper** [comment]
Done in code.

**Comment by BlakeDraper** [comment]
Published with v2.13.3.
